Developers/Git_Basics.rst

```diff
@@ -19,6 +19,13 @@ this will do:
 
     $ sudo apt-get install git-all
```

Comment:
nono, i said in body. shortlog should be Add, body should be Adds :D you mixed them up